package.json

```diff
@@ -21,6 +21,8 @@
         "sulu-security-bundle": "file:src/Sulu/Bundle/SecurityBundle/Resources/js"
     },
     "devDependencies": {
+        "@ckeditor/ckeditor5-dev-utils": "^11.0.1",
+        "@ckeditor/ckeditor5-theme-lark": "^11.1.0",
```

Comment:
These dependencies should probably go as usual dependencies into the `AdminBundle`. This way it does not work if you only install the packages in `sulu-minimal`.